samples/bluetooth/fast_pair/input_device/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -20,13 +20,13 @@
 			reg = <0x100000 DT_SIZE_K(908)>;
 		};
 
-		storage_partition: partition@1e3000 {
-			reg = <0x1e3000 DT_SIZE_K(20)>;
+		bt_fast_pair_partition: partition@1e3000 {
+			label = "bt_fast_pair";
+			reg = <0x1e3000 0x80>;
 		};
 
-		bt_fast_pair_partition: partition@1e8fb8 {
-			label = "bt_fast_pair";
-			reg = <0x1e8fb8 0x48>;
+		storage_partition: partition@1e3080 {
+			reg = <0x1e3080 0x5f80>;
```

Comment:
please check how the storage subsys will behave when there will be such partition that is not divisable by the 0x1000 (i think it was default sector size)

Reply:
So, it says that I have five sectors of 4096 bytes which is 0x5000 in hex. The extra 0xf80 is unused and should be removed I guess. However, if I set the reg to: `reg = <0x1e3080 0x5000>;`, the sample fails to boot. It only works if I set it to `reg = <0x1e4000 0x5000>;` (or `reg = <0x1e3080 0x5f80>;` as it's done now, but then we're wasting the extra 0xf80). @rghaddab, could you expand more on that? Anyway, it seems that we should start from the address divisible by 0x1000. I will set the storage partition to `reg = <0x1e4000 0x5000>;` and the Fast Pair partition to reg = <0x1e3080 0x80>; so that it precedes the storage partition. We will leave the gap between the DFU partition and Fast Pair partition, but I think it's the best we can do.